### DISCUSSION TITLE
Isn't there still a way to fill the "two-door seeders" carrying "seed" and "solid fertilizier"?

### DISCUSSION THREAD

**Discussion by huseyinsayar** [Stephan-S/FS19_AutoDrive#2041] [Q&A]
Isn't there still a way to fill the "two-door seeders" carrying "seed" and "solid fertilizier"?
i want
Go to the field with AD, sow and fertilize with CP, when the seed or fertilizer is gone;
1) If possible, fill seeds and fertilizers from the auger wagons at the edge of the field,
2) If not possible, fill the silos with seeds and fertilizers.
https://resim-yukle.com/hmv4

**Comment by JustDen1234** [comment]
If you line up Seed silo and Fertilizer silo after eachother the seeder can be refilled. I use the Seed Hut and Fertilizer Hut from this mod: https://www.farming-simulator.com/mod.php?lang=en&country=nl&mod_id=129738 and that works for me. 
![image](https://user-images.githubusercontent.com/68871402/129341227-40f9ac8a-ce8a-4c4c-9f36-fb904e2ab5c8.png)

_Note: I'm an AutoDrive user, not one of the programmers._
